client/src/components/Common/EditEvent/EditEvent.js

```diff
@@ -1,38 +1,53 @@
-import React, { Component } from "react";
-import Calender from "../../../Images/calendar_icon.png";
-import Clock from "../../../Images/clock_icon.png";
-import RoomIcon from "../../../Images/room_icon.png";
-import "./editEvent.css";
-import axios from "axios";
-import { Link } from "react-router-dom";
-import Popup from "./Popup";
+import React, { Component } from 'react';
+import Calender from '../../../Images/calendar_icon.png';
+import Clock from '../../../Images/clock_icon.png';
+import RoomIcon from '../../../Images/room_icon.png';
+import './editEvent.css';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import Popup from './Popup';
+import moment from 'moment'
+
 class EditEvent extends Component {
   constructor(props) {
     super(props);
     this.state = {
       userId: 3,
-      date: "2020-05-23T11:00:00.000Z",
-      roomId: 5,
-      room_name: "Moscow",
+      date: '',
+      roomId: null,
+      room_name: '',
       showPopup: false,
-      eventId: "",
-      eventTitle: "",
-      eventAuthor: "",
-      eventDescription: "",
-      eventNote: "",
+      eventId: '',
+      eventTitle: '',
+      eventAuthor: '',
+      eventDescription: '',
+      eventNote: '',
       message: null,
+      rooms: []
     };
   }
 
   componentDidMount() {
     let event = this.props.location.state.event;
+    let date = moment (this.props.location.state.event.event_date.toLocaleString ()).format (
+      'YYYY-MM-DD H:mm:ss'
+    ) 
     this.setState({
       eventId: event.event_id,
       eventTitle: event.event_title,
       eventAuthor: event.event_author,
       eventDescription: event.event_description,
       eventNote: event.event_note,
+      date: date,
+      roomId: event.room_id,
     });
+    axios
+      .get(`/api/rooms`)
+      .then((res) => this.setState({ rooms: res.data })).then(() => {
+        this.state.rooms = this.state.rooms.filter(room => room.room_id === this.state.roomId)
+        this.setState({ room_name: this.state.rooms[0].room_name })
```

Comment:
The room name still an empty

Reply:
no, it display it, but take little time for that